fast_td3/fast_td3/train_utils.py

```
@@ -0,0 +1,268 @@
+        stats[f"{model_name}/grad_{group}_norm"] = all_grads.norm().item()
+    
+    # Overall gradient norm
+    all_params_grads = [p.grad.flatten() for p in model.parameters() if p.grad is not None]
+    if all_params_grads:
+        all_grads = torch.cat(all_params_grads)
+        stats[f"{model_name}/grad_overall_norm"] = all_grads.norm().item()
+    
+    return stats
+
+
+def create_actor(
```

Pylint (4.0.2)

[refactor] Too many arguments (10/5)

(R0913)